packages/browser/rollup.bundle.config.mjs

```diff
@@ -8,6 +8,10 @@ if (targets.some(target => target !== 'es5' && target !== 'es6')) {
   throw new Error('JS_VERSION must be either "es5" or "es6"');
 }
 
+const addonIntegrationFiles = ['contextlines', 'httpclient', 'reportingobserver'];
```

Comment:
let's call this `browserPluggableIntegrationFiles` maybe? A bit more explicit what it does, wdyt?

Reply:
Sounds good, fits better with `coreIntegrationFiles` below 👍🏻